helm/charts/determined/templates/master-config.yaml

```diff
@@ -95,3 +95,48 @@ data:
     {{- if .Values.clusterName }}
     cluster_name: {{ .Values.clusterName }}
     {{- end }}
+
+    {{- if .Values.logging }}
+    logging:
+      {{- if .Values.logging.backend }}
+      backend: {{ .Values.logging.backend }}
+      {{- end }}
+
+      {{- if (eq (default "" .Values.logging.backend) "elastic") }}
+      host: {{ required "A valid host must be provided if logging to Elasticsearch!" .Values.logging.host }}
+      port: {{ required "A valid port must be provided if logging to Elasticsearch!" .Values.logging.port }}
+      {{- if .Values.logging.security }}
+      security:
+        {{- if .Values.logging.security.username }}
+        username: {{ .Values.logging.security.username }}
+        {{- end }}
+        {{- if .Values.logging.security.password }}
+        password: {{ .Values.logging.security.password }}
+        {{- end }}
+        {{- if .Values.logging.security.tls }}
+        tls:
+          {{- if .Values.logging.security.tls.enabled }}
+          enabled: {{ .Values.logging.security.tls.enabled }}
+          {{- end }}
+          {{- if .Values.logging.security.tls.skipVerify }}
+          skip_verify: {{ .Values.logging.security.tls.skipVerify }}
+          {{- end }}
+          {{- if .Values.logging.security.tls.certificate }}
+          certificate: /etc/determined/elastic.crt
+          {{- end }}
+          {{- if .Values.logging.security.tls.certificateName }}
+          certificate_name: {{ .Values.logging.security.tls.certificateName }}
+          {{- end }}
+        {{- end}}
+      {{- end }}
+      {{- end }}
+    {{- end}}
+  {{- if .Values.logging }}
+  {{- if .Values.logging.security }}
+  {{- if .Values.logging.security.tls }}
+  {{- if .Values.logging.security.tls.certificate }}
```

Comment:
question: since these are just go `text/template`'s, can this just be something like `if and .Values.logging .Values.logging.security ...`?

Reply:
`and` in templates doesn't do short-circuiting, so unfortunately not.